packages/contracts/deploy/new/40_finalize-managing-dao/99_verify_step.ts

```diff
@@ -3,19 +3,21 @@ import {HardhatRuntimeEnvironment} from 'hardhat/types';
 
 import {Operation} from '../../../utils/types';
 import {checkPermission, getContractAddress} from '../../helpers';
+import {DAO__factory} from '../../../typechain';
```

Comment:
Unlike the comments above, this seems to be fetching the current version

Reply:
The previous occurrences were wrong and have been fixed in https://github.com/aragon/osx/commit/1ac77f580109dd5c4612e7a70899fae68135606f